.env.example

```
@@ -85,6 +85,17 @@ CORS_EXPOSED_HEADERS=Access-Control-Allow-Origin,Access-Control-Allow-Credential
 CORS_MAX_AGE=3600
 
+# ----------------------------
+# Content Security Policy (CSP) - Frontend
+# ----------------------------
+# These are used by the webapp nginx container for CSP headers.
+# CSP_CONNECT_SRC: Allowed origins for fetch/XHR/WebSocket connections.
+#   Include 'self', your backend API URL, and auth provider (Keycloak).
+#   Example: 'self' https://api.example.com https://auth.example.com
+CSP_SCRIPT_SRC='self' 'unsafe-inline'
+CSP_STYLE_SRC='self' 'unsafe-inline'
+CSP_CONNECT_SRC='self' https://api.example.com https://auth.example.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CSP_CONNECT_SRC key should go before the CSP_SCRIPT_SRC key

(UnorderedKey)